src/decompress.rs

```diff
@@ -484,8 +523,16 @@ impl<'src> DecompressStarted<'src> {
                 let rest: &mut [T] = &mut image_dst[width * start_line..];
                 let rows = (&mut rest.as_mut_ptr()) as *mut *mut T;
 
-                let rows_read = ffi::jpeg_read_scanlines(&mut self.dec.cinfo, rows as *mut *mut u8, 1) as usize;
-                debug_assert_eq!(start_line + rows_read, self.dec.cinfo.output_scanline as usize, "wat {}/{} at {}", rows_read, height, start_line);
+                let rows_read =
+                    ffi::jpeg_read_scanlines(&mut self.dec.cinfo, rows as *mut *mut u8, 1) as usize;
```

Comment:
Please don't break formatting. This project intentionally avoids rustfmt, because rustfmt makes weird mistakes like this line.

Reply:
Sorry about that, my editor is setup to call into RA/rustfmt automatically; I'll probably extract the function I added, reset my branch on top of `master` then just drop it in as a patch.